src/scripts/alert.js

```diff
@@ -17,5 +17,4 @@ export default function createAlert(type, message, title) {
             NotificationManager.error(message, title, ALERT_DELAY_MS);
             break;
       }
-    $(".notification-container").attr("tabindex", -1).focus();
```

Comment:
Will the screen readers read other notifications, created with createAlert() method, when there will be no focus on them?